test/ducks/apps/components/__snapshots__/discover.spec.js.snap

```diff
@@ -868,8 +882,15 @@ exports[`Discover component should display error from props correctly 1`] = `
   <withRouter(Wrapper)
     actionError={null}
     apps={Array []}
+    fetchLatestApp={undefined}
```

Comment:
These stuffs are back beacuse `cozy-scripts` is still using an old version of jest. It will be cleaned at the nexte version.